uit/gui_tools/submit.py

```diff
@@ -184,19 +184,33 @@ def update_file_path(self, _, index):
         else:
             self.env_values[index].value = self.file_browser.value[0]
 
+    def get_versions(self, uit_client):
+        return []
+
     @param.depends('environment_variables')
     def environment_variables_view(self):
         self.environment_variables.pop('', None)  # Clear blank key if there is one
         self.env_names = list()
         self.env_values = list()
         self.env_browsers = list()
         self.env_delete_buttons = list()
+        alert_pane = pn.pane.Str('')
 
         for i, (k, v) in enumerate(self.environment_variables.items()):
-            name_widget = self.env_var_widget(val=k, tag=f'env_key_{i}')
-            val_widget = self.env_var_widget(val=str(v), tag=f'env_val_{i}')
-            browser_widget = self.env_file_browser_widget(tag=f'env_browser_{i}')
-            delete_btn = self.env_delete_btn(tag=f'env_del_{i}')
+            if k == 'HELIOS_VERSION':
```

Comment:
PyUIT is an open source "3rd party" library that is not directly tied to Helios. We therefore can't have any code that refers to any specific software here. It would have to be added in a subclass in application specific code.